ImageDB/vitess_connection.py

```diff
@@ -185,5 +185,58 @@ def insertFeature(self, feature):
 	def insertImagefeatures(self, relations):
 		sql = 'INSERT IGNORE INTO RELATION(Feature_ID, IV_ID) VALUES (%s, %s)'
 		self.mycursor.executemany(sql, relations)
+
+
+	#this function takes a dictionary of arguments and queries the Vitess database 
+	def getImage(self,arguments):
+
+		query = "SELECT * FROM IMAGE_VIDEO INNER JOIN CAMERA ON IMAGE_VIDEO.Camera_ID = CAMERA.Camera_ID WHERE ";
```

Comment:
we can add IN to also filter the feature in this query; it will goes something like 
```
      SELECT *
      FROM FEATURE
    LEFT JOIN 
   (
        SELECT * 
        FROM IMAGE_VIDEO
        JOIN CAMERA
        ON MAGE_VIDEO.Camera_ID = CAMERA.Camera_ID
        WHERE [add your metadata query fields]
    ) AS aNewTable
   ON aNewTable.IV_ID = FEATURE.IV_ID
   WHERE  [add your feature query fields]
```